Makefile

```
@@ -285,6 +285,15 @@ link-check-local: ## Check repository local links with Lychee (offline)
 
 lint-all: lint-kotlin lint-rust lint-android ## Run all linters
 
+sonar: check-tools ## Run local SonarQube analysis (coverage + scan)
```

checkmake (0.3.2)

[warning] Target body for "sonar" exceeds allowed length of 5 lines (7).

(maxbodylength)